Assets/XML/Text/Regions.xml

```diff
@@ -12,303 +12,455 @@
 	<TEXT>
 		<Tag>TXT_KEY_REGION_0</Tag>
 		<English>Britain</English>
-		<French>Britain</French>
+		<French>
```

Comment:
The formatting in this file looks wrong.

Reply:
Not sure why, apparently the kind of tab was wrong. Should be fixed now.